utils.py

```diff
@@ -291,3 +290,142 @@ def plot_mean_node_arity(self, show_counts=False):
         )
         if show_counts:
             plt.bar_label(bars, fmt="{:,.0f}")
+
+    def calc_mutations_per_tree(self):
+        tree_mutations = np.zeros(self.ts.num_trees)
+        for tree in self.ts.trees():
+            tree_mutations[tree.index] = tree.num_mutations
+        return tree_mutations
```

Comment:
I added a comment on https://github.com/tskit-dev/tsinfer-qc/issues/8#issuecomment-1583549582 suggesting an efficient method for calculating mutations per tree.